src/2023/styles/styles.css

```diff
@@ -231,3 +231,25 @@ ul {
 .bold {
   font-weight: bold;
 }
+
```

Comment:
Could we add a comment to delimit the new section here? I believe these rules are not "utilities".

Reply:
Oh, I haven't seen the "Utilities" section... i moved it up and added a new section!